docs/user-guide/administration/backup-restoration.md

```diff
@@ -0,0 +1,158 @@
+# GreptimeDB Export & Import Tools
+
+This guide describes how to use GreptimeDB's Export and Import tools for database backup and restoration.
+
+## Table of Contents
+- [Overview](#overview)
+- [Export Tool](#export-tool)
+- [Import Tool](#import-tool)
+- [Common Usage Scenarios](#common-usage-scenarios)
+- [Best Practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+
+## Overview
+
+The Export and Import tools provide functionality for backing up and restoring GreptimeDB databases. These tools can handle both schema and data, allowing for complete or selective backup and restoration operations.
+
+## Export Tool
+
+### Command Syntax
+```bash
+greptime export [OPTIONS]
+```
+
+### Options
+| Option | Required | Default | Description |
+|--------|----------|---------|-------------|
+| --addr | Yes | - | Server address to connect |
+| --output-dir | Yes | - | Directory to store exported data |
+| --database | No | greptime-* | Name of the database to export |
```

Comment:
please add the syntax about `{my_catalog_name}-{my_database_name}`

Reply:
I'll remove `catalog` since there is only one named `greptime`.